### ISSUE TITLE
Pre build binaries missing for v1.0.4 (OSX + node 7.3.0)

### ISSUE THREAD

**Issue by jariz** [mapbox/node-or-tools#35]
```
npm i

> node_or_tools@1.0.4 install /Users/Jari/Copy/www/node-or-tools
> node-pre-gyp install --fallback-to-build

node-pre-gyp ERR! Tried to download(403): https://mapbox-node-binary.s3.amazonaws.com/node_or_tools/v1.0.4/Release/node-v51-darwin-x64.tar.gz
node-pre-gyp ERR! Pre-built binaries not found for node_or_tools@1.0.4 and node@7.3.0 (node-v51 ABI) (falling back to source compile with node-gyp)
```

It only 403's with darwin, the linux binary does exist.

**Comment by daniel-j-h** [comment]
> node@7.3.0

We let Travis publish binaries for Node.js LTS versions 4 and 6 only, see [here](https://github.com/mapbox/node-or-tools/blob/5bcfa5ddb1f8563bf6ba4b0faadece2486379be0/.travis.yml#L20-L35):

```
$ aws s3 ls s3://mapbox-node-binary/node_or_tools/v1.0.4/Release/
2017-08-07 14:09:09    2693659 node-v46-darwin-x64.tar.gz
2017-08-07 14:07:24    2751871 node-v46-linux-x64.tar.gz
2017-08-07 14:12:30    2693505 node-v48-darwin-x64.tar.gz
2017-08-07 14:07:27    2752019 node-v48-linux-x64.tar.gz
```

**Comment by jariz** [comment]
Fair enough, did not know non-LTS versions weren't supported by you guys.

**Comment by jariz** [comment]
If the building from source gets fixed, then this isn't that much of a problem eitherway

**Comment by daniel-j-h** [comment]
Just published v1.0.5 in which Travis now publishes binaries for Node.js v4,6,8 x Linux,macOS.

**Comment by srinivasdhanabalan** [comment]
Having similar issue with MacOS Monterey v12.6.2, NodeJS version 14. 

node-pre-gyp ERR! Tried to download(403): https://mapbox-node-binary.s3.amazonaws.com/node_or_tools/v1.0.4/Release/node-v83-darwin-arm64.tar.gz
node-pre-gyp ERR! Pre-built binaries not found for node_or_tools@1.1.1 and node@14.20.1 (node-v83 ABI) (falling back to source compile with node-gyp)
Can any one help?
